### PR TITLE
plain text

### PR DESCRIPTION
**Pull request by andiwand**
**Fix**

- #265

### CODE REVIEW

File: include/odr/html.h [hunk 1]
```diff
@@ -73,6 +75,8 @@ struct HtmlPage final {
 };
 
 namespace html {
+Html translate(const TextFile &document, const std::string &path,
```
Comment (TomTasche) [inline include/odr/html.h:78]:
```suggestion
Html translate(const TextFile &text_file, const std::string &path,
```

**Review by TomTasche — COMMENTED** [review]
I guess we could easily add editing for TextFile too?

crazy idea: what if we convert additional formats (like txt and csv) into ODF (odt and ods) and display them like that?

**Comment by andiwand** [comment]
@TomTasche 

the only challenge with editing is how to communicate the diff. the easiest is to pass the whole content of the txt but that's not pretty and wont scale well.

the problem with converting files is that you have to infer information like font and size and paragraphs. later when you want to save the changes into the original format you have to convert backwards and therefore double the chance of breaking something in the document

**Comment by andiwand** [comment]
holding back because it breaks iOS builds

**Pull request merged**